frontend/src/pages/item/_itemId/index.vue

```diff
@@ -272,7 +272,8 @@ export default defineComponent({
   computed: {
     isPlayable(): boolean {
       // TODO: Move this to a mixin
-      return ['PhotoAlbum', 'Photo', 'Book'].includes(this.item.Type as string)
+      return this.item.Type &&
+        ['PhotoAlbum', 'Photo', 'Book'].includes(this.item.Type)
```

Comment:
We have ``BaseItemKind`` now

Reply:
Apologies, I'm not sure what you mean by this comment. This was changed because `Type` could be undefined so it first null checks and then does the array check.

Reply:
@aweebs Basically this, but now I'm realising it doesn't solve the problem:

``[BaseItemKind.PhotoAlbum, BaseItemKind.Photo, BaseItemKind.Photo].includes...``